pkg/routeagent/controllers/route/route.go

```diff
@@ -372,6 +373,8 @@ func (r *Controller) processNextPod() bool {
 					r.podWorkqueue.Forget(obj)
 					return fmt.Errorf("failed to add FDB entry on the Gateway Node vxlan iface %v", err)
 				}
+				klog.Infof("FDB entry added on the Gateway node's vxlan iface for "+
+					"route agent pod %s, IP %s", pod.Name, pod.Status.PodIP)
```

Comment:
I do prefer double-quoting names and general strings...

Reply:
a, skitt clarifies \"%s\" for the pod name. Makes sense